.github/workflows/tests.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       matrix:
         fedora_version:
-          - 40
```

Comment:
We are still releasing to F40. The F40 EOL date is **Tue 2025-05-13**

Reply:
Reminder: The EOL is today. Shall I rebase, anything else needs to be done here? If someone else is taking this over I will happily close the PR. Thanks.

Reply:
Go ahead with rebasing :+1: